src/libraries/PosmLiquidityLibrary.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.24;
+
+import {IPoolManager} from "@uniswap/v4-core/src/interfaces/IPoolManager.sol";
+import {StateLibrary} from "@uniswap/v4-core/src/libraries/StateLibrary.sol";
+import {Position} from "@uniswap/v4-core/src/libraries/Position.sol";
+import {PoolKey} from "@uniswap/v4-core/src/types/PoolKey.sol";
+import {PoolIdLibrary} from "@uniswap/v4-core/src/types/PoolId.sol";
+
+import {PositionConfig} from "./PositionConfig.sol";
+
+/// @notice A library for reading posm liquidity balances from core, wraps core's StateLibrary
+library PosmLiquidityLibrary {
```

Comment:
I do wonder if we should be keeping the name `posm` out of the final code ... i feel like no one else knows what it means?

Reply:
oh also i dont think it needs to be `positionmanager` specific at all? really any contract that adds liquidity could use this library right?
i feel like it should be called `LiquidityLibrary` and `bytes32 salt` should be passed in instead of `tokenId`?
And then its just a generic library for liquidity providing contracts to use for their liquidity look ups?